tools/Dockerfile-tools

```diff
@@ -14,6 +14,5 @@ RUN apt-get update \
         peco \
         vim \
         expect \
-    && curl --http1.1 -L https://cnfl.io/cli | sh -s -- -b /usr/bin v1.18.0 \
-    && curl --http1.1 -L https://cnfl.io/ccloud-cli | sh -s -- -b /usr/bin \
+    && curl --http1.1 -L https://cnfl.io/cli | sh -s -- -b /usr/bin v2.0.0 \
```

Comment:
If CP 7.0.0 is supported only with confluent v1, then it needs to include confluent v1 for on-prem and confluent v2 for ccloud 

cc: @ethanruhe @DABH